paasta_tools/cli/cmds/logs.py

```diff
@@ -422,65 +527,37 @@ def tail_logs(self, service, levels, components, clusters, raw_mode=False):
         implement one of the other options.
         """
-        scribe_envs = set([])
-        for cluster in clusters:
-            scribe_envs.update(self.determine_scribereader_envs(components, cluster))
-        log.info("Would connect to these envs to tail scribe logs: %s" % scribe_envs)
         queue = Queue()
         spawned_processes = []
-        for scribe_env in scribe_envs:
-            # Tail stream_paasta_<service> for build or deploy components
-            if any([component in components for component in DEFAULT_COMPONENTS]):
-                # Start a thread that tails scribe in this env
-                kw = {
-                    'scribe_env': scribe_env,
-                    'stream_name': get_log_name_for_service(service),
-                    'service': service,
-                    'levels': levels,
-                    'components': components,
-                    'clusters': clusters,
-                    'queue': queue,
-                    'filter_fn': paasta_log_line_passes_filter,
-                }
-                process = Process(target=self.scribe_tail, kwargs=kw)
-                spawned_processes.append(process)
-                process.start()
-
-            # Tail Marathon logs for the relevant clusters for this service
-            if 'marathon' in components:
-                for cluster in clusters:
-                    kw = {
-                        'scribe_env': scribe_env,
-                        'stream_name': 'stream_marathon_%s' % cluster,
-                        'service': service,
-                        'levels': levels,
-                        'components': components,
-                        'clusters': [cluster],
-                        'queue': queue,
-                        'parse_fn': parse_marathon_log_line,
-                        'filter_fn': marathon_log_line_passes_filter,
-                    }
-                    process = Process(target=self.scribe_tail, kwargs=kw)
-                    spawned_processes.append(process)
-                    process.start()
-
-            # Tail Chronos logs for the relevant clusters for this service
-            if 'chronos' in components:
-                for cluster in clusters:
-                    kw = {
-                        'scribe_env': scribe_env,
-                        'stream_name': 'stream_chronos_%s' % cluster,
-                        'service': service,
-                        'levels': levels,
-                        'components': components,
-                        'clusters': [cluster],
-                        'queue': queue,
-                        'parse_fn': parse_chronos_log_line,
-                        'filter_fn': chronos_log_line_passes_filter,
-                    }
-                    process = Process(target=self.scribe_tail, kwargs=kw)
-                    spawned_processes.append(process)
-                    process.start()
+
+        def callback(component, stream_info, scribe_env, cluster):
```

Comment:
can you... give this function a better name? I don't have any great suggestions, how about `start_scribe_tail`?